engine/platform/android/dlsym-weak.cpp

```diff
@@ -78,21 +114,75 @@ static unsigned elfhash(const char* _name) {
    Binary Interface) where in Chapter 5 it discuss resolving "Shared
    Object Dependencies" in breadth first search order.
  */
-static Elf_Sym* dlsym_handle_lookup(soinfo* si, const char* name) {
-    return soinfo_elf_lookup(si, elfhash(name), name);
+static Elf_Sym* dlsym_handle_lookup( soinfo* si, const char* name )
+{
+	return soinfo_elf_lookup( si, elfhash( name ), name );
 }
 
-extern "C" void* dlsym_weak(void* handle, const char* symbol) {
+#include "lib_android.h"
+
+static int dladdr_fallback( const void *addr, Dl_info *info )
+{
+	static soinfo *server_info;
+	Elf_Sym *sym;
+
+	if( !server_info )
+		server_info = ( soinfo* )ANDROID_GetServerLibrary();
```

Comment:
You called `ANDROID_` here when the function is called `Android_`.

I think it won't compile.